content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -326,56 +372,17 @@ windows {
 }
 ----
 
-
-=== Defining global variables
-
-Internally, scripts in the `vars` directory are instantiated as a singleton
-on-demand, when used first.  So it is possible to define more methods,
-properties on a single file that interact with each other:
-
-[source,groovy]
-----
-// vars/acme.groovy
-def setFoo(v) {
-    this.foo = v;
-}
-def getFoo() {
-    return this.foo;
-}
-def say(name) {
-    echo "Hello world, ${name}"
-}
-----
-
-Then the Pipeline Script can invoke these functions, rooted off the `acme`
-object:
-
-[source,groovy]
-----
-acme.foo = "5";
-echo acme.foo; // print 5
-acme.say "Joe" // print "Hello world, Joe"
-----
-
-[NOTE]
-====
-A variable defined in a shared library will only show up in _Global Variables
-Reference_ (under _Pipeline Syntax_) after you have first run a successful
-build using that library, allowing its sources to be checked out by Jenkins.
-====
-
-
 === Defining a more structured DSL
 
 If you have a lot of Pipeline jobs that are mostly similar, the global
-function/variable mechanism gives you a handy tool to build a higher-level DSL
+variable mechanism gives you a handy tool to build a higher-level DSL
 that captures the similarity. For example, all Jenkins plugins are built and
 tested in the same way, so we might write a step named
-`jenkinsPlugin`:
+`buildPlugin`:
 
 [source,groovy]
 ----
-// vars/jenkinsPlugin.groovy
+// vars/buildPlugin.groovy
```

Comment:
Why remove the `jenkins`? Wouldn't that make the example clearer wrt what is being built here?